wire-library/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
Is this file necessary?

Reply:
Hmmm, maybe not? I thought I needed one because wire-library is its own project. I'm not sure how spotless look for editorconfig
I did a quick test locally and it seems to apply properly without this.

I'll remove for now